src/applications/bmqtool/m_bmqtool_inpututil.cpp

```diff
@@ -34,12 +35,9 @@ namespace m_bmqtool {
 
 bool InputUtil::getLine(bsl::string* out)
 {
-    const int BUFFER_SIZE = 512;
-    char      buffer[BUFFER_SIZE];
     bsl::cout << "> " << bsl::flush;
```

Comment:
We could also add some ASSERT statements for args in this file which are missing to ptr arguments, to make it consistent with other parts of the codebase. But this could be done in a separate PR